### PR TITLE
Allow the scale control's text to overflow the container

### PR DESCRIPTION
**Pull request by Malvoz**
Fixes the issue described by @secondl1ght in https://github.com/Leaflet/Leaflet/pull/8543:

scale-overflow-hidden

By allowing the scale control's text to overflow the container and giving it a slight `text-shadow`, as suggested in https://github.com/Leaflet/Leaflet/pull/8543#issuecomment-1272492170:

scale-text-overflow-visible

### CODE REVIEW

**Comment by Malvoz** [comment]
The addition of `text-shadow` here results in higher contrast, in which case the need for https://github.com/Leaflet/Leaflet/pull/8547 might not be as immediate. Anyhow, if https://github.com/Leaflet/Leaflet/pull/8547 is merged together with this PR we'd have:

scale-text-overflow-visible2

File: dist/leaflet.css [hunk 1]
```diff
@@ -436,12 +436,10 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	line-height: 1.1;
 	padding: 2px 5px 1px;
 	white-space: nowrap;
-	overflow: hidden;
 	-moz-box-sizing: border-box;
 	     box-sizing: border-box;
-
-	background: #fff;
 	background: rgba(255, 255, 255, 0.8);
+  text-shadow: 1px 1px #fff;
```
Comment (mourner) [inline dist/leaflet.css:442]:
```suggestion
	text-shadow: 1px 1px #fff;
```

Comment (Malvoz) [reply dist/leaflet.css:442]: Oops.

**Review by mourner — APPROVED** [review]

**Comment by secondl1ght** [comment]
Thanks for fixing this so quickly, will this be included in the next release?

**Comment by Falke-Design** [comment]
@secondl1ght yes in the v2

**Pull request merged**